fix: console devtools service exposure (#159)
AmadeusITGroup/otter: docs/dev-tools/console-devtools.md

# Otter Console DevTools

Otter Framework is exposition function to the browser console to help to debug and configure your application.
All the helpers are exposed under the variable **_OTTER_DEVTOOLS_** accessible via `window._OTTER_DEVTOOLS_`.

> Currently the following modules are exposing helpers in the console:

Reviewer: ```suggestion
Currently the following modules expose helpers in the console:
```

> - **Configuration**: display and update the application configuration

Reviewer: ```suggestion
- **Configuration**: displays and updates the application configuration
```

> - **Localization**: toggle localization key visualization

Reviewer: ```suggestion
- **Localization**: toggles localization key visualization
```

> - **RulesEngine**: display rules engine status information

Reviewer: ```suggestion
- **RulesEngine**: displays rules engine status information
```

> - **Application**: display application global information

Reviewer: ```suggestion
- **Application**: displays application global information
```

## How to enable the support in your application

You will need to enable the features you need one by one. There is no root toggle.
To do so, you will need to import the corresponding modules in you AppModule:

```typescript

import { ApplicationDevtoolsModule } from '@o3r/application';
import { ConfigurationDevtoolsModule } from '@o3r/configuration';
import { LocalizationDevtoolsModule } from '@o3r/localization';
import { RulesEngineDevtoolsModule } from '@o3r/rules-engine';

@NgModule({
  imports: [
    ApplicationDevtoolsModule,
    ConfigurationDevtoolsModule,
    LocalizationDevtoolsModule,
    RulesEngineDevtoolsModule
  ]
})
export class AppModule {
}

```

> Then the services activation can be done in the AppComponent as following:

Reviewer: ```suggestion
Then the services activation can be done in the AppComponent as follows:
```

```typescript
import { ApplicationDevtoolsConsoleService } from '@o3r/application';
import { ConfigurationDevtoolsConsoleService } from '@o3r/configuration';
import { LocalizationDevtoolsConsoleService } from '@o3r/localization';
import { RulesEngineDevtoolsConsoleService } from '@o3r/rules-engine';

@Component({
  selector: 'app'
})
export class AppComponent {
  constructor(
    applicationDevtoolsConsoleService: ApplicationDevtoolsConsoleService,
    configurationConsoleService: ConfigurationDevtoolsConsoleService,
    localizationConsoleService: LocalizationDevtoolsConsoleService,
    rulesEngineDevtoolsConsoleService: RulesEngineDevtoolsConsoleService) {
    this.getStaticConfig();
    if (environment.DEBUG_MODE) {

>       // The COnsole Devtools services should be activated only in the development mode

Reviewer 2: ```suggestion
      // The Console Devtools services should be activated only in the development mode
```
